holoviews/element/raster.py

```diff
@@ -517,13 +517,18 @@ class ImageStack(Image):
     - overlay/ndoverlay of Images
     """
 
+    vdims = param.List(doc="""
```

Comment:
I do not include `bounds=(0, None)` here because it errors without a default, but I also don't want the default to be `z` because it only plots the first layer.